docs/UserGuide.md

```diff
@@ -171,11 +192,16 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from SocialBook.
+
+* All entries even those not currently shown in the list will be cleared.
```

Comment:
Just a grammatical suggestion here: "All entries, including those not currently shown in the list, will be cleared"